package.json

```diff
@@ -117,6 +117,16 @@
           }
         ],
         "priority": "option"
+      },
+      {
+        "viewType": "one.viewer.relation",
+        "displayName": "Relation Viewer",
+        "selector": [
+          {
+            "filenamePattern": "*"
+          }
+        ],
+        "priority": "option"
```

Comment:
Could you make this package.json file changes into a separate PR? 
Because the backend logics are not yet merged, it will show a dummy result. Therefore, let's merge this package.json later when the backend data is ready.